launcher/ui/MainWindow.cpp

```diff
@@ -45,100 +45,97 @@
 #include "MainWindow.h"
```

Comment:
Any changes in this file except for reformatting?

Reply:
not anymore